zxing-android-embedded/src/com/google/zxing/client/android/BeepManager.java

```diff
@@ -94,23 +94,20 @@ public synchronized void playBeepSoundAndVibrate() {
 
     public MediaPlayer playBeepSound() {
         MediaPlayer mediaPlayer = new MediaPlayer();
-        mediaPlayer.setAudioStreamType(AudioManager.STREAM_MUSIC);
-        mediaPlayer.setOnCompletionListener(new MediaPlayer.OnCompletionListener() {
-            @Override
-            public void onCompletion(MediaPlayer mp) {
-                mp.stop();
-                mp.release();
-            }
+        mediaPlayer.setAudioAttributes(new AudioAttributes.Builder().setContentType(
+                AudioAttributes.CONTENT_TYPE_MUSIC).build());
```

Comment:
Fixes warning: W/MediaPlayer: Use of stream types is deprecated for operations other than volume control

Reply:
This is causing crashes in Android 4 devices(4.4.2 and 4.4.4). 
[AudioAttributes](https://developer.android.com/reference/android/media/AudioAttributes) was added in API level 21. 

StackTrace: 
```
Fatal Exception: java.lang.NoClassDefFoundError: android.media.AudioAttributes$Builder
       at com.google.zxing.client.android.BeepManager.playBeepSound(BeepManager.java:97)
       at com.google.zxing.client.android.BeepManager.playBeepSoundAndVibrate(BeepManager.java:84)
       at com.journeyapps.barcodescanner.CaptureManager$1.barcodeResult(CaptureManager.java:80)
       at com.journeyapps.barcodescanner.DecoratedBarcodeView$WrappedCallback.barcodeResult(DecoratedBarcodeView.java:50)
       at com.journeyapps.barcodescanner.BarcodeView$1.handleMessage(BarcodeView.java:52)
       at android.os.Handler.dispatchMessage(Handler.java:98)
       at android.os.Looper.loop(Looper.java:136)
       at android.app.ActivityThread.main(ActivityThread.java:5584)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1268)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1084)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

Reply:
@ANewGalaxy @rkistner